manifest_prod.yml

```diff
@@ -1,12 +1,16 @@
 ---
 memory: 512M
 stack: lucid64
+instances: 2
+env:
+  NEW_RELIC_CONFIG_FILE: newrelic.ini
+  NEW_RELIC_ENV: staging
 applications:
 - name: api
   path: .
   host: fec-prod-api
   env:
-    PRODUCTION: True
+    FEC_API_DEBUG: False
```

Comment:
Agreed that enforcing a standard prefix should happen. Flask-appconfig looks good. The only thing I'd like to see is a more declarative method for listing expected config variables, types, defaults, etc., which is what I like about argparse-like libraries (although we can store default values in a config file with flask-appconfig too). People also seem to like Click: http://click.pocoo.org/4/options/#values-from-environment-variables.

Anyway, I think that basically any approach to standardizing environment parsing is a good idea, so let's go with whichever looks best to you @arowla.